ports/plplot/vcpkg.json

```
@@ -1,12 +1,20 @@
 {
   "name": "plplot",
   "version-string": "5.13.0",
```

```suggestion
  "version-semver": "5.13.0",
```
This should be version-semver. https://github.com/PLplot/PLplot/blob/6a12ffe24060803e1d66ae457fa7663dc4ded5bf/cmake/modules/plplot_version.cmake#L22-L23